nurseconnect/forms.py

```diff
@@ -0,0 +1,101 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+
+from molo.profiles import forms as molo_profiles_forms
+from molo.profiles.models import UserProfile
+
+
+class RegistrationForm(forms.Form):
+    username = forms.RegexField(
+        regex=r'^[\w.@+-]+$',
+        widget=forms.TextInput(
+            attrs=dict(
+                required=True,
+                max_length=30,
+            )
+        ),
+        label=_("Username"),
```

Comment:
This label might not even be needed since its overridden by the fed styling. Checking with @CSergienko on this.